### ISSUE TITLE
[DISCOVERY] Build foundational domiliciary knowledge 

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#24704]
## Issue Description
VA Domiciliaries are VA health facilities which are included as locations under the relevant VAMC. They don't fit the facility pattern established by the VAMC facility detail pages as they include minimal services, such as "extended care" (or none at all). The information that Veterans need for these types of facilities may be different than that which is expected for medical clinics. 

### Domiciliaries

Domiciliaries are "shelters" providing services to homeless Veterans (or to returning service members who may not have a home in the area to which they have returned). They can also provide shelter to rehab patients or victims of domestic violence.  

There are currently 5 locations in dual-state status:
- Philadelphia
- Eastern Colorado (has 2)
- Houston
- Jackson, MS

**Examples**
Domiliciary example: [Philadelphia](https://www.va.gov/philadelphia-health-care/locations/)

<details>
<summary> Screenshot </summary>

![image](https://user-images.githubusercontent.com/55411834/115441894-a1833e80-a1ce-11eb-812f-0fe02cd7b8b3.png)

</details>

### Problem to be solved
Before we can reimagine detail pages or validate potential designs with Veterans, we need to better understand these facilities. 

### Questions
  - how is the facility best described?
  - is the facility free-standing? (vs a floor of a hospital, for example) 
  - what services are provided?
  - what is the referral process? do Veterans self-refer?
  - what information does a Veteran need for these types of facilities?
  - Who is the likely audience? Is it a Veteran, caregiver, or other advocates?
  - What info is meaningful for the "prepare for your visit" section? Is there specific eligibility criteria?

## Tasks
- [ ] Prepare basic conversation guide for conversations with SMEs (plan to interview director and/or clinician from Eastern Colorado, Jackson and Houston. 
- [ ] Interview SME to gain foundational understanding about these facilities, including the intended audience for detail pages

## Acceptance Criteria
- [ ] Document information from SME conversations

**Comment by mmiddaugh** [comment]
@nicko-amida there may also be a facility (or two) in MN: https://mn.gov/mdva/assets/2019-08-doms-brochure_tcm1066-396897.pdf

**Comment by nicko-amida** [comment]
@mmiddaugh Ok thanks

**Comment by mmiddaugh** [comment]
Background info: 
[Domiciliary Care for Homeless Veterans Program](https://www.va.gov/homeless/dchv.asp)
[Definition of Domiciliary Care](https://www.federalregister.gov/documents/2018/04/06/2018-07082/definition-of-domiciliary-care)

**Comment by mmiddaugh** [comment]
[Puget Sound VA - American Lake Intensive PTSD facility](https://www.seattletimes.com/seattle-news/renovated-veterans-facility-offers-intensive-ptsd-care-tranquillity-trout/)

**Comment by mmiddaugh** [comment]
<details>
<summary> Documenting my research notes here </summary> 



VA domiciliaries are generally defined as "a temporary home" for Veterans

**eligibility** 
- no income beyond military pension/no adequate means of support 
- may have chronic disability or disease preventing them from earning a living
- not able to live independently but not requiring acute inpatient/medical or nursing home admission

In 2010, the Domiciliary Care programs and Psychosocial Rehabilitation Treatment Programs (PRRTP) were merged into a single system of residential care to become MH RRTPs.

**Mental Health Residential Rehabilitation Treatment Program (MHRRTP)** 
- comprehensive treatment and rehab for mental health or substance use disorders and medial/psychosocial needs (homelessness/unemployment)
- Veterans can self-refer or be referred by VA or community programs

MHRRTP includes the following 24/7 residential models: 
 - Domiciliary Residential Rehabilitation Treatment Programs - larger residential programs with multiple units for various populations
 - Domiciliary Care for Homeless Veterans Program
- Health Maintenance Domiciliary Beds Program - focuses on symptom reduction and stabilization to facilitate community integration
- General Domiciliary or Psychosocial Residential Rehabilitation Treatment Program
- Domiciliary Substance Abuse Programs (Dom SA)
- Domiciliary Post-Traumatic Stress Disorder Programs (Dom PTSD) - includes MST
   -  a minimum of 4 hours of therapeutic programming/day (7 days a week) 
   - ranges from short-term care of limited focus, (e.g., less than 30 days and targeted primarily towards diagnosis-specific treatment and symptom management) to long-term, comprehensive rehabilitation (e.g., more than 30 days with a full-range of psychosocial services, such as life-skills training, social learning, vocational rehabilitation therapy, etc.)
  - open to active duty

**Women Veterans**
- New residential programs or renovations to existing space must support a 20 percent minimum utilization rate for women Veterans
- must have access to female clinicians 5 days/wk

**SUD**
- Over 85 percent of Veterans admitted to an MH RRTP have a Substance Use Disorder (SUD) diagnosis
- SUD units must offer at least 2 evidence based interventions, including CBT

**Compensated Work Therapy-Transitional Residence (CWT-TR)**
- referrals: VA inpatient and outpatient programs, self-referrals, community, mail-in applications, and referrals from the Homeless Programs
- stay is generally 12 mos or less
- designed for peer support and self-care, minimal staffing
- goal is transition to community
</details>

**Comment by mmiddaugh** [comment]
Notes from Meeting with Lisa on 7/7 
Program =  "package of services"

Domiliciaries
- Most Doms are aligned with only one VAMC system but may appear listed only under the main facility. 
- More likely to be freestanding
- Recommends Butler, Coatesville and San Diego as potential interviews
- Domiliciary not an official national health service

**Comment by mmiddaugh** [comment]
[Email for SME](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/facilities/domiciliaries)
